sigs/sig-marketing/README.md

```diff
@@ -0,0 +1,17 @@
+# Argo Sig Marketing
```

Comment:
This is great progress. Here are some additional ideas.
- It is critical to call out the responsibilities, boundaries, rules for the marketing effort here.
- The marketing group should be to promote the Argo Open Source brand.
- I understand all the marketing professionals are from vendors, as the contributors are mainly developers. So, everyone including us will face the balance and conflict every now and then between the vendor initiative and open source. So I recommend limiting the vendor max seats to 2 as membership to have a balanced conversation.
- As more vendors are behind the curtain now and building the association with the argo product. I am pretty sure such interests could come and go based on the vendor's true interest level. It is crucial to call out the Sig Marketing responsibilities to ensure the longevity of the Argo brand, rather than a springboard.